tools/sof-kernel-log-check.sh

```diff
@@ -28,8 +28,10 @@ else
 fi
 
 if [ "$err" ]; then
-    echo "Caught error: $err"
-    echo "kernel log has error(s)"
+    echo `date -u '+%Y-%m-%d %T %Z'` "[ERROR]" "Caught dmesg error"
+    echo "===========================>>"
+    echo $err
```

Comment:
please use `echo "$err"`, it will keep string format

Reply:
sure